tests/test_beamtime.py

```diff
@@ -0,0 +1,322 @@
+
+"""
+unit tests for beamtime info
+"""
+
+import datetime
+import os
+import socket
+import subprocess
+import sys
+import unittest
+
+_test_path = os.path.dirname(__file__)
+_path = os.path.join(_test_path, '..')
+if _path not in sys.path:
+    sys.path.insert(0, _path)
+
+from apstools.beamtime import bss_info, bss_info_makedb
+
+
+BSS_TEST_IOC_PREFIX = "ioc:bss:"
+
+
+def using_APS_workstation():
+    hostname = socket.gethostname()
+    return hostname.lower().endswith(".aps.anl.gov")
+
+
+def bss_IOC_available():
+    import epics
+    # try connecting with one of the PVs in the database
+    cycle = epics.PV(f"{BSS_TEST_IOC_PREFIX}esaf:cycle")
+    cycle.wait_for_connection(timeout=2)
+    conn = cycle.connected
```

Comment:
Codacy found an issue: [local variable 'conn' is assigned to but never used (F841)](https://app.codacy.com/gh/BCDA-APS/apstools/pullRequest?prid=5764884)